integration-tests/oidc-wiremock-providers/src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+quarkus.http.auth.proactive=false
+quarkus.log.category."org.htmlunit".level=ERROR
+quarkus.log.category."com.github".level=ERROR
+quarkus.keycloak.devservices.enabled=false
+
+quarkus.oidc.slack.provider=slack
+quarkus.oidc.slack.auth-server-url=${keycloak.url}/slack
+quarkus.oidc.slack.client-id=7925551513107.7922794171477
+quarkus.oidc.slack.credentials.secret=2b82d6039bc97946460fdec75fadd9b2
+quarkus.oidc.slack.authentication.extra-params.team=quarkus-oidc-slack-demo
+quarkus.oidc.slack.tenant-paths=/providers/slack*
+quarkus.oidc.slack.token.lifespan-grace=2147483647
+quarkus.oidc.slack.authentication.force-redirect-https-scheme=false
+quarkus.oidc.slack.authentication.verify-access-token=false
+quarkus.oidc.slack.authentication.remove-redirect-parameters=false
+quarkus.oidc.slack.authentication.scopes=oidc,profile,email
```

Comment:
I am bit lost, you asked me to remove `profile` and `email` from `io.quarkus.oidc.runtime.providers.KnownOidcProviders#slack` because users may not need it. I thought you were right, but now I am getting lost. If I actually set `ret.getToken().setRequiredClaims(Map.of("profile", "email"));` in there, according to the documentation they will be required. So I think we should not set them?

Reply:
So I have actually dropped all these scopes (line you are commenting on), so they are gone. I am not going to preset them for users because I have tested it and these scopes are not required to be granted to your Slack application. I don't remember if they are actually on ID token. Maybe I misunderstood your initial request.

Reply:
Hi Michal @michalvavrik, I only asked you to avoid pre-setting them during the Slack application registration, sorry for the confusion, I'll restore scopes, and will merge, thanks

Reply:
@michalvavrik I've done 0 work here, thanks for driving this PR to the completion